src/BarComponent/BarComponent.ts

```diff
@@ -107,9 +107,9 @@ class BarComponent {
       .attr('class', '.bar-group')
       .attr('value', d => d.value);
 
-    const bars = this.renderBars(shapes as NodeSelection, groupedx, y);
-    this.renderUncertainty(bars as NodeSelection, groupedx, y);
-    this.renderLabels(bars as NodeSelection, groupedx, y);
+    const bars = this.renderBars(shapes as unknown as NodeSelection, groupedx, y);
```

Comment:
Can you explain why additional `unknown` type cast is required here?

Reply:
The typescript update lead to the typescript compiler complaining here.